content/en/docs/release_notes/wicklow.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Wicklow release notes
+linkTitle: Wicklow
+description: 30 September 2022
+date: 2022-09-30
+Hide_readingtime: true
+---
+## Summary
+Fixed a bug with `@axway/requester` on calculating the accurate `Content-Length` of the request `body`.

```

Comment:
**issue:** comment made regarding this summary on the v5 docs and unresolved.

Reply:
@alasdairhurst , So is it ok to keep the Summary and Fixes doc same?

Reply:
Fixed